easybuild/tools/package/package_naming_scheme/easybuild_pns.py

```diff
@@ -50,4 +49,4 @@ def version(self, ec):
             # cfr. http://rpm.org/ticket/56,
             # https://debian-handbook.info/browse/stable/sect.manipulating-packages-with-dpkg.html (see box in 5.4.3)
             ebver.replace('dev', '~dev')
```

Comment:
I'm not expert here but shouldn't we somehow concat this with the change below so the function returns either `4.x.x~eb` or `4.x.x~ebdev`...or is it ok to have 2 `~`?

Reply:
Good information to clarify and analyze it may be here: https://www.debian.org/doc/debian-policy/ch-controlfields.html#version
As I understand, having two tildas is allowed. And in the case of having `4.x.x~eb` and `4.x.x~dev~eb`, the first one seems to be the latest

Reply:
To be honest, I didn't check such form for the case of rpm packages

Reply:
For me then this seems ok, but maybe @boegel should give the final ok